CHANNELS.md

```diff
@@ -0,0 +1,46 @@
+# Channels
+
+`fuelup` adopts a simplified version of `rustup` [channels](https://rust-lang.github.io/rustup/concepts/channels.html). Currently, only the `latest` channel will be published and serve as a source of distribution of Fuel toolchain binaries.
+
+The `latest` channel is possibly updated by a scheduled GitHub workflow that **runs every 30 minutes** and checks for new releases of `forc` and `fuel-core`.
+
+### How the CI updates the `latest` channel
+
+A channel is automatically updated if integration tests pass for the specific version of `forc` and `fuel-core`. This channel is typically found in a TOML file, and for the `latest` channel, is named `channel-fuel-latest.toml`.
+
+### How the workflow works, a summary
+
+The short explanation of how the scheduled workflow works is that we fetch all the latest versions after the latest published versions within `channel-fuel-latest.toml`, and we run tests for the possible combinations of them if not already marked as incompatible. After tests are done we either mark the version sets as incompatible if the tests failed or update the channel.
```

Comment:
These "How the CI updates the `latest` channel" and "How the workflow works, a summary" sections sound a little rambly and could possibly be structured a little better.

How about something along these lines:


> # Channels
> 
> *Introduce the concept of channels...*
> 
> ### `latest`
> 
> The `latest` channel is `fuelup`'s default channel. It provides access to the latest compatible, published releases of `forc` and `fuel-core`.
> 
> When installing the `latest` channel, `fuelup` will refer to the `channel-fuel-latest.toml` file within this repository to determine the set of `forc` and `fuel-core` versions to retrieve. The versions in this file are updated by a scheduled GitHub workflow that runs once every 30 minutes and performs the following steps:
> 
> 1. Checks for newly published versions of `forc` and `fuel-core`.
> 2. Tests compatibility of new versions against a set of integration tests.
> 3. Selects the latest set of versions that successfully pass the tests.
> 4. Publishes the selected versions to the `channel-fuel-latest.toml` manifest.
> 
> 
> 
> ### `nightly`
> 
> *TODO*
>
> ---

Reply:
I added the `nightly` section with a bit more explanation that we'll support it soon.